docs/rfc9948.html

```
@@ -0,0 +1,816 @@
+<dt>その他の情報:</dt>
+<dd>
+<a href="https://datatracker.ietf.org/doc/rfc9948">Datatracker</a>|<a href="https://www.rfc-editor.org/info/rfc9948">Info page</a>
+</dd>
+</dl></div>
+<div class="row">
+<div lang="en" class="col">
+<span></span><h1 id="rfcnum">RFC 9948</h1>
+<h1 id="title">Internet Protocol Police (IPP) - Schedule of Punishments</h1>
+</div>
+<div lang="ja" class="col">
+<span></span><h1 id="rfcnum">RFC 9948</h1>
```

HTMLHint (1.9.2)

[error] The id value [ rfcnum ] must be unique.

(id-unique)